src/main/webapp/WEB-INF/jsp/settings/settings_form.jsp

```diff
@@ -514,6 +517,12 @@
                                                 <spring:message code="settings.absenceTypes.action.approve.label" />
                                             </label>
                                         </td>
+                                        <td data-th-text="<spring:message code='settings.absenceTypes.table.head.color' />">
+                                            <label for="color-${loop.index}" class="tw-sr-only">
+                                                <spring:message code="settings.absenceTypes.action.color.label" />
+                                            </label>
+                                            <input type="color" id="color-${loop.index}" name="absenceTypeSettings.items[${loop.index}].color" value="${absenceType.color}">
```

Comment:
What colors do we want to allow? As far as I understand, selection of any possible color could lead to issues about visibility. So we need a predefined set of colors. How many? We have 15 different VacationTypes (could be extended) maybe coloring based on the VacationCategory (5) could be enough? 

As mentioned in the corresponding issue, the following color set should be working for light and dark theme:

> Vielleicht ein funktionierendes Set von Farben: ![image](https://user-images.githubusercontent.com/142107/161221323-c50912cc-3b38-4cb1-9f0b-3d1c6f6612af.png)

I have no idea how to build a good color picker component. Maybe something like [this](https://codepen.io/nmcinteer/pen/owZoqe)? The html color picker is not [possible to limit](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/color#browser_compatibility) in certain browsers (e.g. firefox).

Reply:
radio buttons with the same `name` attribute (so only one is selectable)

like:

```
<label><input type="radio" name="color" value="blue">Blue</label>
<label><input type="radio" name="color" value="green">Green</label>
```

aligned in a css grid.

Reply:
Maybe Grafana Style? 
![grafik](https://user-images.githubusercontent.com/142107/166814664-8fa3db6c-63a2-4421-a03b-fca7d04fd100.png)

Reply:
The color red could be reserved for sicknotes

Reply:
https://tailwindcss.com/docs/customizing-colors#default-color-palette

* neutral
* red
* orange
* yellow
* lime
* emerald
* cyan
* blue
* violet
* fuchsia